### ISSUE TITLE
config file path is wrong in the rendering process

### ISSUE THREAD

**Issue by florian** [nathanbuchar/electron-settings#9]
In the main process `settings.getConfigFilePath()` is `/Users/florian/Library/Application Support/MYAPPNAME/electron-settings/settings.json`. In the renderer process (web page) it's `/Users/florian/Dropbox/code/MYAPPNAME/node_modules/electron-settings/lib/electron-settings/settings.json`.

I suppose this is because `require('app').getPath('userData')` doesn't work in the web page, so the library probably falls back on something else.

**Comment by nathanbuchar** [comment]
Interesting, I should add a note to the README that this package should be used outside of the render process. Can't say I've ever used it within the render process, so I've never come across this, so thanks for bringing it to my attention.

I have a busy week, but I'll look into this is soon as I have some free time.

Thanks :-)

**Comment by florian** [comment]
> this package should be used outside of the render process

Why is that though? I think it really makes the library a lot less useful.
I'd like to separate the logic of handling those settings (main process) with the UI for the user to change those settings (renderer process).

Currently it's possible to still use the same settings file in the renderer process by doing this:
```javascript
var ElectronSettings = require('electron-settings');
var path = require('remote').require('app').getPath('userData');
var settings = new ElectronSettings({configDirPath: path});
```

However those settings won't be in sync with all the `ElectronSettings` instances in the main/renderer processes. I assume this is because of the cache and because the events aren't shared across all processes. So I can't let the user change the settings in the UI and watch for those changes in the main process.
Either way I currently can't get around sending all settings changes through [`ipc`](https://github.com/atom/electron/blob/master/docs/api/ipc-main.md) myself if I want a settings UI.

**Comment by nathanbuchar** [comment]
This should be fixed now in 1.0.3 :)
